src/redux/actions/index.js

```diff
@@ -58,4 +58,11 @@ export const showAddressWindow = address => ({
 export const hideAddressWindow = () => ({
   type: ACTION_TYPES.HIDE_ADDRESS_WINDOW,
   payload: ''
-});
\ No newline at end of file
+});
+
+export const getGeolocation = location  => {
+  return {
+    type: ACTION_TYPES.GET_GEOLOCATION,
+    payload: location
+  }
+};
```

Comment:
For syntax consistency, can we write this the same way as the other action creators above?